python-sdk/nuscenes/eval/tracking/README.md

```diff
@@ -0,0 +1,298 @@
+* At inference time users may use all past sensor data and ego poses from the current scene, but not from a previous scene. At training time there are no restrictions.
+
+### General rules
+* We release annotations for the train and val set, but not for the test set.
+* We release sensor data for train, val and test set.
+* Users make predictions on the test set and submit the results to our evaluation server, which returns the metrics listed below.
+* We do not use strata. Instead, we filter annotations and predictions beyond class specific distances.
+* Users must limit the number of submitted boxes per sample to 500.
+* Every submission provides method information. We encourage publishing code, but do not make it a requirement.
+* Top leaderboard entries and their papers will be manually reviewed.
+* Each user or team can have at most one one account on the evaluation server.
+* Each user or team can submit at most 3 results. These results must come from different models, rather than submitting results from the same model at different training epochs or with slightly different parameters.
+* Any attempt to circumvent these rules will result in a permanent ban of the team or company from all nuScenes challenges.
+
+## Results format
+We define a standardized tracking result format that serves as an input to the evaluation code.
+Results are evaluated for each 2Hz keyframe, also known as `sample`.
+The tracking results for a particular evaluation set (train/val/test) are stored in a single JSON file. 
+For the train and val sets the evaluation can be performed by the user on their local machine.
+For the test set the user needs to zip the single JSON result file and submit it to the official evaluation server.
+The JSON file includes meta data `meta` on the type of inputs used for this method.
+Furthermore it includes a dictionary `results` that maps each sample_token to a list of `sample_result` entries.
+Each `sample_token` from the current evaluation set must be included in `results`, although the list of predictions may be empty if no object is tracked.
+```
+submission {
+    "meta": {
+        "use_camera":   <bool>  -- Whether this submission uses camera data as an input.
+        "use_lidar":    <bool>  -- Whether this submission uses lidar data as an input.
+        "use_radar":    <bool>  -- Whether this submission uses radar data as an input.
+        "use_map":      <bool>  -- Whether this submission uses map data as an input.
+        "use_external": <bool>  -- Whether this submission uses external data as an input.
+        "conf_thresh":  <floor> -- Confidence threshold which determines whether a track is positive or negative.
+    },
+    "results": {
+        sample_token <str>: List[sample_result] -- Maps each sample_token to a list of sample_results.
+    }
+}
+```
+For the predictions we create a new database table called `sample_result`.
+The `sample_result` table is designed to mirror the `sample_annotation` table.
```

Comment:
It might be a good idea to give a link to `sample_annotation` table, so that the first time users might know the schema.